DataManagementSystem/Service/StorageElementHandler.py

```diff
@@ -50,6 +50,39 @@
 MAX_STORAGE_SIZE = 0
 USE_TOKENS = False
 
+def diskSpace(path, size = 'TB', total = False):
+    """
+      Returns disk usage of the given path.
+      If no size is specified, terabytes will be used by default.
+      If total is set to true, the total disk space will be returned instead.
+    """
+
+    if size.upper() == "KB":
+      convert = 1024
+    elif size.upper() == "MB":
+      convert = 1.048576e6
+    elif size.upper() == "GB":
+      convert = 1.073741824e9
+    elif size.upper() == "TB":
+      convert = 1.099511627776e12
+    else:
+      return S_ERROR( "No valid size specified" )
+
+    try:
+      st = os.statvfs(path)
+
+      if total:
+        # return total space
+        result = (st.f_blocks * st.f_frsize) / convert
+      else:
+        # return free space
+        result = (st.f_bavail * st.f_frsize) / convert
+
+    except Exception as e:
```

Comment:
of course it does, but it is a bad practice to use a catch all if you can have a finer grain selection :) Pylint would tell you as well ;-)